.gitignore

```diff
@@ -60,3 +60,7 @@ ehthumbs.db
 # Folder config file
 Desktop.ini
 /samples/ChangeFeedProcessor/.vs/DocumentDB.ChangeFeedProcessor/v15/sqlite3/storage.ide
+/samples/ChangeFeedProcessor/.vs/DocumentDB.ChangeFeedProcessor/v15/Server/sqlite3/storage.ide-wal
```

Comment:
nit: Should we add the entire `.vs` folder as ignored?

Reply:
good idea. will do that.